x-pack/plugins/cases/public/components/all_cases/cases_metrics.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent, useEffect, useMemo } from 'react';
+import {
+  EuiDescriptionList,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLoadingSpinner,
+  EuiToolTip,
+} from '@elastic/eui';
+import styled, { css } from 'styled-components';
+import prettyMilliseconds from 'pretty-ms';
+import { CaseStatuses } from '../../../common/api';
+import { useGetCasesStatus } from '../../containers/use_get_cases_status';
+import { StatusStats } from '../status/status_stats';
+import { useGetCasesMetrics } from '../../containers/use_get_cases_metrics';
+import { MTTR_DESCRIPTION, MTTR_STAT } from './translations';
+
+interface CountProps {
+  refresh?: number;
+}
+const MetricsFlexGroup = styled.div`
+  ${({ theme }) => css`
+    .euiFlexGroup {
+      border: ${theme.eui.euiBorderThin};
+      border-radius: ${theme.eui.euiBorderRadius};
+      margin: 0 0 ${theme.eui.euiSizeL} 0;
+    }
+    @media only screen and (max-width: ${theme.eui.euiBreakpoints.s}) {
+      .euiFlexGroup {
+        padding: ${theme.eui.euiSizeM};
+      }
+    }
+  `}
+`;
+
+export const CasesMetrics: FunctionComponent<CountProps> = ({ refresh }) => {
+  const {
+    countOpenCases,
+    countInProgressCases,
+    countClosedCases,
+    isLoading: isCasesStatusLoading,
+    fetchCasesStatus,
+  } = useGetCasesStatus();
+
+  const { mttr, isLoading: isCasesMetricsLoading, fetchCasesMetrics } = useGetCasesMetrics();
+
+  const mttrValue = useMemo(
+    () => (mttr ? prettyMilliseconds(mttr * 1000, { compact: true, verbose: false }) : 'N/A'),
```

Comment:
Could you use a dash `-` here rather than `N/A`? Otherwise `N/A` should be internationalized.